kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -939,15 +1100,15 @@ define([
                 model.setItem('firstLine', message.logs.first + 1);
                 model.setItem('lastLine', message.logs.first + viewLines.length);
                 model.setItem('totalLines', message.logs.max_lines);
-                render();
+                renderLog();
             }
             if (looping) {
-                scheduleNextRequest();
+                scheduleLogRequest();
             }
         }
 
         function startEventListeners() {
-            let ev = runtime.bus().listen({
+            listenersByType['job-logs'] = bus.listen({
```

Comment:
I put these into an object so it's easier to remove listeners if they are not needed any more